.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: '[Feature request] '
+labels: enhancement
```

Comment:
If it is a feature request it should have the feature-request label. Of course, this goes together with splitting enhancement and feature-request.